pkg/minikube/perf/binary.go

```diff
@@ -70,10 +114,9 @@ func (b *Binary) download() error {
 	}
 	defer client.Close()
 
-	// first make sure object exists
-	obj := client.Bucket(bucket).Object(fmt.Sprintf("%d/minikube-%s-amd64", b.pr, runtime.GOOS))
-	if _, err := obj.Attrs(ctx); err != nil {
-		return fmt.Errorf("minikube binary for pr %v does not exist in bucket", b.pr)
+	obj, err := newObjectFromStorage(ctx, client, b.pr)
```

Comment:
Why was this code moved into a new function? It doesn't seem like there is any difference between `newObjectFromStorage` and the code that was here before